src/Symfony/Component/HttpKernel/Attribute/Cache.php

```diff
@@ -13,6 +13,10 @@
 
 /**
  * Describes the default HTTP cache headers on controllers.
+ * Headers defined in with the Cache attribute are ignored if they are already set
```

Comment:
`defined in with the Cache attribute` -> `defined in the Cache attribute`

Reply:
```suggestion
 * Headers defined in the Cache attribute are ignored if they are already set
```